File: chatmessageview/src/main/java/com/github/bassaer/chatmessageview/utils/DateFormatter.java
package com.github.bassaer.chatmessageview.utils;

import java.util.Calendar;

/**
 * Date formatter of chat timeline separator.
 * Created by nakayama on 2017/01/13.
 */
public class DateFormatter implements ITimeFormatter {
    @Override
    public String getFormattedTimeText(Calendar createdAt) {
        return TimeUtils.calendarToString(createdAt, "MMM. dd, yyyy");
    }
}

File: chatmessageview/src/main/kotlin/com/github/bassaer/chatmessageview/util/DateFormatter.kt
package com.github.bassaer.chatmessageview.util

import com.github.bassaer.chatmessageview.utils.TimeUtils
import java.util.*

/**
 * Date formatter of chat timeline separator.
 * Created by nakayama on 2017/01/13.
 */
class DateFormatter : ITimeFormatter {
    override fun getFormattedTimeText(createdAt: Calendar): String {
        return TimeUtils.calendarToString(createdAt, "MMM. dd, yyyy")
    }
}
